FWCore/ParameterSet/src/types.cc

```diff
@@ -1058,11 +1167,30 @@ bool edm::encode(std::string& to, std::vector<std::string> const& from) {
   return true;
 }  // encode from vector<string>
 
+bool edm::encode_element(std::string& to, std::string const& from) { return encode(to, from); }
+
+bool edm::encode_deprecated(std::string& to, std::vector<std::string> const& from) {
+  to = "{";
+
+  std::string converted;
+  for (std::vector<std::string>::const_iterator b = from.begin(), e = from.end(); b != e; ++b) {
+    if (!encode_deprecated(converted, *b)) {
```

Comment:
How about
```suggestion
  for (auto const& b : from) {
    if (!encode_deprecated(converted, b)) {
```
?

Reply:
For the initial code change, I was just trying to minimize the changes. Can do what suggested.

Reply:
Ah right, missed that, thanks.